### PR TITLE
Update Bookinfo code

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**
Today, after TLS external services are supported without necessary TLS origination, https://istio.io/blog/2018/egress-https/ must be rewritten to show access with TLS origination and without it. The code of the details must be changed accordingly.

### CODE REVIEW

File: samples/bookinfo/kube/bookinfo-details-v2.yaml [hunk 1]
```diff
@@ -34,6 +34,6 @@ spec:
         ports:
         - containerPort: 9080
         env:
-        - name: WITH_ISTIO
+        - name: DO_NOT_USE_TLS_FOR_EXTERNAL_SERVICE_CALLS
```
Comment (rshriram) [inline samples/bookinfo/kube/bookinfo-details-v2.yaml]: Shouldn’t we be setting this to false always to show SNI routing ? 
Also this name is too big. I suggest use_https_googleapis which is short n sweet

Comment (vadimeisenbergibm) [reply samples/bookinfo/kube/bookinfo-details-v2.yaml]: @rshriram I propose to use this example to demonstrate both HTTP with TLS origination and HTTPS, and to compare between them. The idea is to demonstrate an organization that requires monitoring of the content and requires not to use TLS (HTTP with TLS origination). The environment variable simulates some general policy variable to enforce the policy of the organization. Maybe a simpler name would be `DO_NOT_USE_TLS` or `DO_NOT_ENCRYPT`.
Then I will compare this approach with HTTPS - an organization that does not have strict monitoring requirements and is OK to perform monitoring by SNI.

**Comment by andraxylia** [comment]
@vadimeisenbergibm can we do this on release-1.0 branch?

That will get merged periodically.

**Comment by vadimeisenbergibm** [comment]
@andraxylia yes, let me resubmit the PR to release-1.0 branch.

**Comment by vadimeisenbergibm** [comment]
Resubmitted as https://github.com/istio/istio/pull/6988 to release-1.0.

**Pull request closed without merging**